packages/intrepid2/src/Orientation/Intrepid2_OrientationTools.hpp

```diff
@@ -262,7 +262,7 @@ namespace Intrepid2 {
           \param  cellTopoKey     [in]  - key of the cell topology of the parameterized domain (1- and 2-subcells)
           \param  cellOrt         [in]  - cell orientation number (zero is aligned with shards default configuration
       */
-      template<typename pointType>
+      template<typename DeviceType, typename pointType>
```

Comment:
Why do we need this KOKKOS_INLINE_FUNCTION ? The function interface does not seem to allow this on device as it uses cell topology. This function is a host only function. It also means that this does not require device type.

Reply:
I don't need it to be KOKKOS_INLINE_FUNCTION, but it can be, and I used it before on device. It uses the topology Key,  not the topology. We do this all the time.